vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -203,19 +203,13 @@ spec:
         type: object
     served: true
     storage: false
-status:
```

Comment:
Do you know why regenerating deletes this?

Reply:
I think this was accepted as a bug some time ago: https://github.com/kubernetes-sigs/controller-tools/issues/456 and fixed from [version 0.9.0](https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.9.0) with https://github.com/kubernetes-sigs/controller-tools/pull/630